.github/workflows/zowe-cli-plugin.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x]
```

Comment:
now that node14 is gone, we can remove the `update npm` step since node16 comes with npm8